fail2ban/server/database.py

```diff
@@ -201,6 +216,11 @@ def __init__(self, filename, purgeAge=24*60*60):
 						": updated from '%i' to '%i'",
 						Fail2BanDb.__version__, version, newversion)
 					raise RuntimeError('Failed to fully update')
+
+			# pypy: set journal mode after possible upgrade db:
+			if pypy:
+				cur.execute("PRAGMA journal_mode = MEMORY")
+
```

Comment:
Won't this mean that if pypy is true and db does not exist yet that "PRAGMA journal_mode = MEMORY" will not be done?
I think you need less indentation here